CMakeLists.txt

```diff
@@ -85,6 +85,10 @@ if(USE_CUDA)
   enable_language(CUDA)
 endif()
 
+if(USE_ROCM)
+  enable_language(HIP)
```

Comment:
HIP language is only available since CMake-3.21(see https://cmake.org/cmake/help/v3.21/command/enable_language.html), so please make `USE_ROCM` conditional on 3.21 CMake version

Reply:
The need for `enable_language(HIP)` was removed in the latest commit.